### ISSUE TITLE
Remove Wit's End from Rotations

### ISSUE THREAD

**Issue by DoomRobotBoy** [OvercastNetwork/Rotations#260]
So I know that this map is quite new to the rots, since it was first introduced in GotW, and for some maps it therefore needs time to develop strategies for good matches. But... unfortunately I have not seen such strategies. 

The TNT spawner in the center is barely used for strategic moves (although these are public matches) but instead for teamkilling, teamgriefing (blowing up team bridges) and then for mindlessly spamming tnt around the map. Haven't seen it being used for defending or attacking.

The Center wool room is not a place to defend, of course on T-matches defenses can develop better, because there is more time between the start and the first 'attack', because the wool room is accessed from 3 sides, 1 being lead from the middle, basically directly reached from the TNT spawner, and players can just walk in, only having to destroy a quartz and nether brick block. No one even tries to defend, but placing a wall in front of it, because there are no good defenses which have the time of being made, because there is no time for them.

More of an extra point, because other certain maps have similar unidentified locations:
The golden blocks/armor supply, is an area most probably won't even know about, because its above spawn, of course there are the adventurous who want to see what every map holds, but not everyone is that exploring. The map needs to have a clear indication of where what is, which is on Wit's End not very clear concerning the armor.

TL;DR
- TNT from TNT spawner is used for only bad events
- Center Wool Room is a wool room, so badly located (for public matches) which doesn't encourage anyone to defend it, because of being close to the front-lines and well being in the middle lane.
  (- Armor-location is pretty unclear for the unexperienced player)

Things that could improve it's gameplay:
- Remove TNT spawner and add a spawner like on Golden Drought III (with speed and haste?)
- Remove center wool room, and remove the back lane going past it (also add bedrock on the side lanes leading to the other wool rooms?)
  (- Redo spawn, there are tons of spawn designs, won't be that hard to think of a new effective one.)

The map is original, has unique gameplay and an unique kind of block use, which could work out into a great combination. The way things are currently handled are just not the best.

Please remove if not improved.
